### ISSUE TITLE
MySQL error 42000 on new database

### ISSUE THREAD

**Issue by quamok** [IgnisDa/ryot#165]
When setting up a new MySQL database and pointing Ryot to it, when the Ryot container comes up, I see the following error in the Ryot container log and Ryot shuts down. 

When using sqlite, it's working fine.

```
07/03/2023 9:52:02 AM
2023-07-03T13:52:02.051095Z  INFO ryot: Running version 1.5.2
07/03/2023 9:52:02 AM
2023-07-03T13:52:02.131580Z  INFO ryot: Using database backend: "MySQL"
07/03/2023 9:52:02 AM
2023-07-03T13:52:02.327421Z  INFO sea_orm_migration::migrator: Applying all pending migrations
07/03/2023 9:52:02 AM
2023-07-03T13:52:02.412079Z  INFO sea_orm_migration::migrator: Applying migration 'm20230410_000001_create_metadata'
07/03/2023 9:52:03 AM
2023-07-03T13:52:03.056481Z  INFO sea_orm_migration::migrator: Migration 'm20230410_000001_create_metadata' has been applied
07/03/2023 9:52:03 AM
2023-07-03T13:52:03.078912Z  INFO sea_orm_migration::migrator: Applying migration 'm20230417_000002_create_user'
07/03/2023 9:52:03 AM
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Exec(SqlxError(Database(MySqlDatabaseError { code: Some("42000"), number: 1101, message: "BLOB, TEXT, GEOMETRY or JSON column 'preferences' can't have a default value" })))', apps/backend/src/main.rs:133:35
07/03/2023 9:52:03 AM
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Container stopped
```

**Comment by IgnisDa** [comment]
What version of MySQL are you using?

**Comment by quamok** [comment]
mysql  Ver 8.0.33 for Linux on x86_64 (MySQL Community Server - GPL)

**Comment by IgnisDa** [comment]
I pushed a fix on `v1.5.6`.
